src/main/java/pikappi/command/Command.java

```diff
@@ -0,0 +1,19 @@
+package pikappi.command;
```

Comment:
I like how there are separate packages within the pikappi package to distinguish command classes, task classes and the exception class!